### ISSUE TITLE
Please add APK to release assets

### ISSUE THREAD

**Issue by nomadfarmer** [ccrama/Slide#3237]
Slide version: 6.4
Android version: 8.0

I use f-droid as often as possible, but for some projects it lags significantly. When that happens for a project I use daily (for instance, with newpipe), I prefer to switch to manually updating from github before switching to the play store, but that doesn't seem possible with slide.

For slide, f-droid "recommends" 6.3 for some reason and the latest available version is 6.4.  Investigating an issue I've been having with v.reddit videos not including audio, I checked that f-droid had me up to date. It didn't, but had a more recent version it wasn't switching to (as mentioned above, f-droid was on 6.3 from Feb, but had an upgrade to 6.4 [dated June 6 in f-droid] available... but that's still not up to date compared to the repo or the play store).

tl;dr: I know f-droid can be laggy. Please include a compiled APK in release assets so I can keep up to date without using google play.

**Comment by TacoTheDank** [comment]
@nomadfarmer If you install apps using F-Droid, you'll have to wait until Slide gets updated there (because they build and sign the apps themselves). We can make F-Droid builds, but the signing key will be different (so you'll have to backup your settings, uninstall Slide, and then reinstall the new APK with the different signing key).

I do support this idea though. Only @ccrama has the signing key, so he'll have to do this.

**Comment by obar** [comment]
Time will tell how this goes but I believe that from now on, after [very recent changes](https://gitlab.com/fdroid/fdroiddata/-/merge_requests/7114) , an f-droid build will kick off within a day of a release being tagged and the result will be available very soon after that. No more long waits.

**Comment by TacoTheDank** [comment]
@obar Thanks!
